### ISSUE TITLE
Error 500 when displaying the services page on the mobile version

### ISSUE THREAD

**Issue by Flaburgan** [diaspora/diaspora#5312]
Steps to reproduce:

Go to the /services page.
Click on the "toggle mobile" link in the footer

```
Started GET "/services" for 83.157.17.132 at 2014-10-08 22:19:51 +0200
Processing by ServicesController#index as HTML
Completed 500 Internal Server Error in 5.8ms

ActionView::MissingTemplate (Missing template services/index, application/index with {:locale=>[:fr, :en], :formats=>[:mobile], :handlers=>[:erb, :builder, :markerb, :haml]}. Searched in:
  * "/home/diaspora/diaspora/app/views"
  * "/home/diaspora/.rvm/gems/ruby-2.0.0-p481@diaspora/gems/rails_admin-0.4.9/app/views"
  * "/home/diaspora/.rvm/gems/ruby-2.0.0-p481@diaspora/gems/kaminari-0.15.1/app/views"
  * "/home/diaspora/.rvm/gems/ruby-2.0.0-p481@diaspora/gems/devise-3.2.4/app/views"
):
  lib/rack/chrome_frame.rb:39:in `call'
  lib/unicorn_killer.rb:35:in `call'
```

**Comment by Flaburgan** [comment]
I'm working on this issue for both privacy and services routes.

**Comment by svbergerem** [comment]
Fixed by #6086.
